env_settings.py

```diff
@@ -0,0 +1,109 @@
+                        "type": "custom",
+                        "tokenizer": "keyword",
+                        "filter": ["lowercase"]
+                    }
+                }
+            }
+        }
+    },
+    # Default settings for aliases on all environments (overrides default settings)
+    'hqdomains': {
+        "settings": {
+            "analysis": {
+                "analyzer": {
+                    "default": {
+                        "type": "custom",
+                        "tokenizer": "whitespace",
+                        "filter": ["lowercase"]
+                    },
+                    "comma": {
+                        "type": "pattern",
+                        "pattern": "\s*,\s*"
+                    },
+                }
+            }
+        }
+    },
+
+    'hqapps': {
+        "settings": {
+            "analysis": {
+                "analyzer": {
+                    "default": {
+                        "type": "custom",
+                        "tokenizer": "whitespace",
+                        "filter": ["lowercase"]
+                    },
+                }
+            }
+        }
+    },
+
+    'hqusers': {
+        "settings": {
+            "analysis": {
+                "analyzer": {
+                    "default": {
+                        "type": "custom",
+                        "tokenizer": "whitespace",
+                        "filter": ["lowercase"]
+                    },
+                }
+            }
+        }
+    },
+
+    # Default settings for aliases per environment (overrides default settings for alias)
+    'production': {
+        'xforms': {
+            'settings': {
+                'number_of_shards': 10,
+                "analysis": {
+                    "analyzer": {
+                        "default": {
+                            "type": "custom",
+                            "tokenizer": "whitespace",
+                            "filter": ["lowercase"]
+                        },
+                        "sortable_exact": {
+                            "type": "custom",
+                            "tokenizer": "keyword",
+                            "filter": ["lowercase"]
+                        }
+                    },
+                },
+            },
+        },
+        'hqcases': {
+            'settings': {
+                'number_of_shards': 10,
+                "analysis": {
+                    "analyzer": {
+                        "default": {
+                            "type": "custom",
+                            "tokenizer": "whitespace",
+                            "filter": ["lowercase"]
+                        },
+                        "sortable_exact": {
+                            "type": "custom",
+                            "tokenizer": "keyword",
+                            "filter": ["lowercase"]
+                        },
+                    },
+                },
+            },
+        }
+    },
+}
```

Comment:
Need to add ICDS to this as well. With the new infra we'll have 4 ES nodes. Not sure how many shards we want.